io.github.brunoherbelin.Vimix.json

```diff
@@ -0,0 +1,159 @@
+{
+    "app-id": "io.github.brunoherbelin.Vimix",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "vimix",
+    "rename-desktop-file": "vimix.desktop",
+    "rename-icon": "vimix",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--device=shm",
+        "--device=all",
+        "--filesystem=host",
+        "--filesystem=/tmp",
+        "--talk-name=org.gtk.vfs.*",
+        "--env=FREI0R_PATH=/app/lib/frei0r-1"
+    ],
+    "cleanup": [
+        "/include",
+        "/share/gdb",
+        "/share/info",
+        "/share/man",
+        "/share/doc",
+        "/lib/pkgconfig",
+        "/lib/debug",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "glfw",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DGLFW_BUILD_EXAMPLES=OFF",
+                "-DGLFW_BUILD_TESTS=OFF",
+                "-DGLFW_BUILD_DOCS=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/glfw/glfw/releases/download/3.3.8/glfw-3.3.8.zip",
+                    "sha256": "4d025083cc4a3dd1f91ab9b9ba4f5807193823e565a5bcf4be202669d9911ea6"
+                }
+            ]
+        },
+        {
+            "name": "x264",
+            "config-opts": [
+                "--enable-shared"
+            ],
+            "sources": [
+            {
+                "type": "git",
+                "commit": "baee400fa9ced6f5481a728138fed6e867b0ff7f",
+                "url": "https://code.videolan.org/videolan/x264.git"
+            }
+            ]
+        },
+        {
+            "name": "srt",
+            "buildsystem": "cmake",
```

Comment:
```suggestion
            "buildsystem": "cmake-ninja",
```

Reply:
and this?

Reply:
vimix use the SRT protocol for network I/O streaming
Gstreamer provided in package does not include SRT gst-bad-plugin (srtsink and srtsrc)
It is required that the SRT library is installed before the related gst-bad-plugin can be compiled.

PS: Thanks for your careful review. Please note I started with an empty package before iteratively adding libraries one by one and testing the flatpak locally until all featured could be supported.

Reply:
Tested and compiling with cmake-ninja